src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss

```diff
@@ -16,12 +16,13 @@
     background-image: url("../../resource/img/#{$icon}");
     background-size: 32px 24px;
     background-repeat: no-repeat;
-
+    margin-bottom: 4px;
   }
   .icon+* {
     align-self: center;
     text-transform: uppercase;
     font-size: $font-size-smaller;
+    font-weight: 600;
```

Comment:
let's use @extend .strong which we altered to be font-weight bold. im worried about leaving these littered about to cleanup

Reply:
let me know if you are handling it. or if you can paste the syntax that will be great.

Reply:
done.